README.md

```diff
@@ -58,6 +58,27 @@ my-teradata-db-profile:
       logmech: LDAP
 ```
 
+### Logdata
+
+The logon mechanism for Teradata jobs that dbt executes can be configured with the `logdata` configuration in your Teradata profile. Addtional data like secure token, distinguished Name, or a domain/realm name can be set in your Teradata profile using `logdata`. `logdata` is not used with the TD2 mechanism.
+
+```yaml
+my-teradata-db-profile:
+  target: dev
+  outputs:
+    dev:
+      type: teradata
+      host: <host>
+      schema: dbt_test
+      tmode: ANSI
+      logmech: LDAP
+      logdata: 'authcid=username password=password'
+      port: <port>
+```
+
+For more information on authentication options, go to [Teradata JDBC Driver Reference](https://downloads.teradata.com/doc/connectivity/jdbc/reference/current/frameset.html)
```

Comment:
It might be confusing for users why a python library references documentation for a java library. Have you checked if there is a language-agnostic doc for logmech/logadata we can point to? If such a doc doesn't exist, then I think we should offer an explanation why this references docs for a java lib.

Reply:
Added reference for Teradata Vantage authentication documentation for reference.